pypeit/reduce.py

```diff
@@ -9,8 +9,7 @@
 
 from pypeit import ginga, utils, msgs, processimages, specobjs
 from pypeit.core import skysub, extract, trace_slits, pixels, wave
-
-from IPython import embed
+import IPython
```

Comment:
Is there a reason you prefer this to `from IPython import embed`?
I've been using the latter.  We should choose one or the other.

Reply:
It seems to me like IPython.embed()is slightly more clear.

Reply:
Definitely more clear but 2x more to type.  And I am typing this alot.